js/launcher.js

```diff
@@ -0,0 +1,39 @@
+/* global BaseModule */
+'use strict';
+
+(function(exports) {
+  /**
+   * This module is responsible to read all boot up related
+   * configurations and asks other launcher to launch
+   * at the correct timing by the configurations.
+   */
+  var Launcher = function() {};
```

Comment:
This file is the very first file that could be customized by different form factors. We might move this out of jgyos `system` in future implementation, but I think its good enough for the first minimum viable product.